scripts/ecosystem-ci/before-test.test.ts

```diff
@@ -0,0 +1,23 @@
+import { describe, expect, it } from 'vitest';
```

Comment:
Can you come up with a more descriptive name for this file?

We do not run this file/test as part of ecosystem-ci, we ran it as part of our own CI, correct?
So I'd say a more descriptive name would be `existing-resolutions.test.ts`